R/Figures.R

```diff
@@ -841,7 +841,7 @@ TADA_Scatterplot <- function(.data, id_cols = c("TADA.ComparableDataIdentifier")
 #' # Creates a scatterplot including the two specified sites in the same plot:
```

Comment:
New params need to be added in function description. Will there be a list of allowable values for the new params? Or can the user enter any column name as a group?